packages/devtools_shared/lib/src/server/file_system.dart

```diff
@@ -76,9 +76,95 @@ class LocalFileSystem {
     return jsonEncode(json);
   }
 
+  /// The location of the Flutter store file, ~/.flutter.
+  static String flutterStoreLocation() {
```

Comment:
Is this used anywhere else? If not, it might just be cleaner to keep it the way it was.

If you want to keep this factored out, it should probably be a static getter.